src/main/java/address/model/Person.java

```diff
@@ -240,4 +240,8 @@ public String toString() {
         return "Person: " + getFullName();
     }
 
+    public String getPersonGitHubProfilePageUrl(){
```

Comment:
`getProfilePageUrl` will do. This method is already in `Person` and we only have 1 kind of URL now.